src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasures.ts

```diff
@@ -124,6 +158,8 @@ export function migrateCalculatedMeasures(
             record;
         }
       });
+      // The calculated measure was successfully migrated.
+      counters.calculated_measures.success++;
```

Comment:
We should wrap all the block starting with `migrateCalculatedMeasureContent` with a `try/catch`.
If there is an error, then we increment the failures counter and add a the error to the report.

You will need the `filesAncestry`, `folderName` and `folderId` in this case as well, so you might want to define higher up than they currently are.